examples/bootstrap-script/README.md

```diff
@@ -0,0 +1,116 @@
+# Bootstrapping your users
+
+Before spawning a notebook to the user, it could be useful to 
+do some preparation work in a bootstrapping process.
+
+One common use case is at follows:
+
+* You are using an LDAPAuthenticator and have no home directory for your user on the host
+* You are using a DockerSpawner and would like to mount a volume, but as no directory exists, docker will create one.
+However, as the docker daemon is running as root, the on-the-fly generated directory for the volume mount will not be
+writeable by user jovyan inside of the container
+
+Another use case could be that every newly spawned notebook should come with initial content that you'd like to 
```

Comment:
Perhaps:

*Prepopulating Content:* Another use would be to copy initial content, such as tutorial files or reference material, into the user's space when a notebook server is newly spawned.